src/components/NFTOriginVerifier.tsx

```diff
@@ -195,6 +211,56 @@ export default function NFTOriginVerifier() {
           }
         }
       })();
+    } else if (
+      lookupChain === CHAIN_ID_APTOS &&
+      lookupCreatorAddress &&
+      lookupCollectionName &&
+      lookupTokenName
```

Comment:
Probably we could create something like a `const` boolean on the top to match this validation as:
` const thisIsMySemanticBooleanName:boolean = lookupChain === CHAIN_ID_APTOS && lookupCreatorAddress && lookupCollectionName && lookupTokenName;`

```suggestion
      thisIsMySemanticBooleanName
```

Reply:
fixed